### DISCUSSION TITLE
What is the expected status of react-native-reanimated v2 in RNW?

### DISCUSSION THREAD

**Discussion by pegasusjcrawford** [microsoft/react-native-windows#10433] [Q&A] [answered]
I'm trying to find out whether or not the 2.x releases of react-native-reanimated should be expected to work with RNW? This [pull request](https://github.com/software-mansion/react-native-reanimated/pull/1000) in their repository would seem to imply that it should work with web parity, but when I try to run my app with any 2.x release of the library I hit the following failure:

```
 ERROR  TypeError: Cannot read property 'installCoreFunctions' of undefined, js engine: hermes
```

Would love to be able to incorporate some of the awesome Reanimated v2 tutorial's I've seen in my Windows app 😁

**Comment by chrisglein** [comment, marked as answer]
It's not implemented for Windows. You can test that by going to React Native Directory and doing a [search for the Module](https://reactnative.directory/?search=react-native-reanimated). You'll note that there are tags (and ability to filter) for platform, including Windows.

**Reply by pegasusjcrawford** [reply]
Thanks for linking that, I wasn't aware of the directory! 

If that's the state for the package then shouldn't it be removed from the [react-native-windows docs list of community modules](https://microsoft.github.io/react-native-windows/docs/supported-community-modules#community-modules)? It feels a bit misleading to say it's a community supported module when there isn't currently an "official" path to install a supported version.

**Reply by chrisglein** [reply]
Yeah, I can see how that table is misleading. It's intended to show there isn't complete support, but its inclusion in the table at all may send the wrong message. I've opened a doc bug: https://github.com/microsoft/react-native-windows-samples/issues/723
